akka/templates/autoscaling.yaml

```diff
@@ -12,6 +12,8 @@ metadata:
     {{- end }}
   name: "{{ template "akka.fullname" . }}"
 spec:
+  behavior:
+    {{- toYaml .Values.autoscaling.behavior | nindent 4 }}
```

Comment:
@cw-sakamoto 
Does it work with versions of kubernetes prior to `1.18`?
`Capabilities.KubeVersion` or `Capabilities.APIVersions.Has` may be required.

Reply:
@cw-ozaki 
Sorry, I didn't consider that...